scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -154,13 +192,16 @@ def main():
     if not GITHUB_TOKEN:
         raise RuntimeError('GITHUB_TOKEN does not appear to be set')
 
+    if not args.versions_to_scan:
+        args.versions_to_scan = versions_to_scan
```

Comment:
What's this for? I don't see `args` passed to another fuction and I don't see usage of `args.versions_to_scan`

Reply:
I think this is a vestige of an earlier implementation plan that I didn't follow..

Reply:
Actually, it was mistakenly trying to replace the defaults with the values the user gave us, but I either overwrote the wrong variable or passed the wrong variable to all of the functions that need it below.